src/components/data-table/DataTablePagination.tsx

```diff
@@ -0,0 +1,47 @@
+import React from "react";
+import Pagination from "@material-ui/core/TablePagination";
+
+import { TablePagination } from "./types";
+
+export interface DataTablePaginationProps {
+    pagination: TablePagination;
+    onChange?(newPagination: TablePagination): void;
+}
+
+export function DataTablePagination(props: DataTablePaginationProps) {
+    const { pagination, onChange = () => {} } = props;
+
+    const handleChangePage = (_event: unknown, page: number) => {
+        onChange({
+            ...pagination,
+            page: page + 1,
+        });
+    };
+
+    const handleChangePageSize = (event: React.ChangeEvent<HTMLInputElement>) => {
+        const pageSize = parseInt(event.target.value, 10);
+        onChange({
+            ...pagination,
+            pageSize,
+            page: 1,
+        });
+    };
+
+    return (
+        <Pagination
+            rowsPerPageOptions={pagination.pageSizeOptions}
+            component="div"
+            count={pagination.total}
+            rowsPerPage={pagination.pageSize}
+            page={pagination.page - 1}
+            backIconButtonProps={{
```

Comment:
Maybe to constant outside the function, we will also gain some vertical space.

Reply:
Yeah, not sure if we want to keep the a11y props or not... We do not have them everywhere, I just mirrored the ones that material had in their examples.